corehq/form_processor/submission_post.py

```diff
@@ -162,6 +163,9 @@ def run(self):
                 if instance.xmlns == DEVICE_LOG_XMLNS:
                     try:
                         process_device_log(self.domain, instance)
+                    except ProtocolError:
+                        # if riak is down/struggling, the caller will catch and notify
```

Comment:
Not it's to prevent it from hitting the `except Exception:` clause below it. (Python only handles any exception once, in the first matching `except` clause.)